docs/visualize-data-with-grafana.rst

```diff
@@ -0,0 +1,222 @@
+.. _visualize-data-with-grafana:
+
+===========================
+Visualize data with Grafana
+===========================
+
+`Grafana`_ is an open-source tool that helps you build real-time dashboards, 
+graphs, and all sorts of data visualizations. It is the perfect complement 
+to CrateDB, which is purpose-built for monitoring large volumes of machine 
+data in real-time.
+
+For the purposes of this guide, it is assumed that you
+have a cluster up and running and can access the Console. If not, please refer
+to the :ref:`tutorial on how to deploy a cluster for the first time
+<cloud-tutorials:cluster-deployment>`.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _grafana-load-dataset:
+
+Load a sample dataset
+=====================
+
+To visualize data with Grafana, a dataset is needed first. In this sample, demo
+data is added directly via the CrateDB Cloud Console. To import the data go to the Overview page of your deployed cluster. 
```

Comment:
Please wrap conforming to style guide

Reply:
Ah, right.. good catch. I was playing with auto-wrapper throughout work on this. Must have left it on soft-wrap.